src/js/main.js

```diff
@@ -48,7 +55,36 @@ class Site {
       _this.$covervidVideo.coverVid(1920, 1080);
     }
   }
+
+  submitAgeForm() {
+    $('#submit-age').on('click', function(e){
+      e.preventDefault();
+      const month = $('.month').val();
+      const day = $('.day').val();
+      const year = $('.year').val();
```

Comment:
For these elements we should use more descriptive classes like `.birthday-month`, `.birthday-day`, `.birthday-year`